__tests__/app.test.js

```diff
@@ -58,6 +58,7 @@ describe("GET:/api/articles/:article_id", () => {
         expect(article).toHaveProperty("created_at");
         expect(article).toHaveProperty("votes");
         expect(article).toHaveProperty("article_img_url");
+        expect(article).toHaveProperty("comment_count")
```

Comment:
We shouldn't be edited an earlier test to pass this later ticket. Could you create a new test please? 🙂